tox.ini

```diff
@@ -31,6 +31,10 @@ commands = pylint -f colorized -E google test --ignore=src-gen
 deps = pylint
 commands = pylint -f colorized -d all -e W -r n google test --ignore=src-gen
 
+[testenv:pylint-no-test-deps]
+deps = pylint
```

Comment:
At least locally, `requirements.txt` seems to be completely unnecessary -- all the environments build fine without it. It looks like `tox` is smart enough to install the package via `setup.py` (and pulling in the dependencies in the process), so we don't need that file.

I'm now running into build errors, but it looks like the failure is happening during `brew update`, before `tox` is ever invoked. I'm not sure why; I will look more tomorrow.